src/components/composite/StatusBottomSheet/StatusBottomSheet.stories.tsx

```
@@ -0,0 +1,154 @@
+  render: (args) => {
+    const [open, setOpen] = useState(false);
+
+    useEffect(() => {
+      setOpen(true);
+    }, []);
+
+    return <StatusBottomSheet {...args} isOpen={open} />;
+  },
+};
+
+export const Error: Story = {
```

Biome (2.3.14)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)